typescript/src/generator.ts

```diff
@@ -83,8 +89,6 @@ export class Generator {
     const api = this.buildAPIObject();
     await this.processTemplates(api);
```

Comment:
the `processTemplate` function take basePath, we only give api here, does this effect anything? I'm wondering how we give the path to template since we don't have default `basePath` value in `processTemplate`. Thanks!

Reply:
It's my bad naming. This line calls `this.processTemplates()` (a method in the class), but we also have `import { processTemplates } from './templater'` which is a different function (that takes 2 arguments).

Reply:
Oh, my bad, I just looked into the changing part, not the whole file, thanks for the clarification!